docs/user-guide/cli-arguments.md

```diff
@@ -22,6 +22,7 @@ They are set in the container spec of the `ingress-nginx-controller` Deployment
 | `--disable-sync-events` | Disables the creation of 'Sync' Event resources, but still logs them |
 | `--dynamic-configuration-retries` | Number of times to retry failed dynamic configuration before failing to sync an ingress. (default 15) |
 | `--election-id`                    | Election id to use for Ingress status updates. (default "ingress-controller-leader") |
+| `--election-ttl`                  | Duration a leader election is valid before it's getting re-elected. (Default: 30s) |
```

Comment:
i think here we need to call out the specific way to set this, otherwise they will not add the correct value, something like 

```suggestion
| `--election-ttl`                  | Duration for a leader election is valid before it's getting re-elected. (Default: 30s). 
The syntax in X<unit>, with <unit>: s, h, d |
```